jobs/notebooks/train_presence_model.scala

```diff
@@ -0,0 +1,131 @@
+// Databricks notebook source
+// MAGIC %md
+// MAGIC # Example job
+// MAGIC This notebook will create an example dataset
+
+// COMMAND ----------
+
+import com.databricks.dbutils_v1.DBUtilsHolder.dbutils
+import com.quby.demo.databricks.storage.DeltaRepository
+import com.quby.demo.databricks.util.JobUtils
+import org.apache.spark.sql.SparkSession
+
+// COMMAND ----------
+
+// DBTITLE 1,Input variables
+dbutils.widgets.text("environment", "")
+dbutils.widgets.text("jobName", "")
+dbutils.widgets.text("utcDate", "")
+dbutils.widgets.text("days", "1")
+dbutils.widgets.text("rawDb", "staging")
+dbutils.widgets.text("featureDb", "")
+
+val environment = dbutils.widgets.get("environment")
+val jobName = dbutils.widgets.get("jobName")
+val days = dbutils.widgets.get("days").toInt
+val rawDb = dbutils.widgets.get("rawDb")
+val featureDb = dbutils.widgets.get("featureDb")
+
+val utcDateTo = JobUtils.getOrTodayUTC(dbutils.widgets.get("utcDate"))
+val utcDateFrom = JobUtils.subtractDays(utcDateTo, days)
+
+implicit val spark = SparkSession.builder().getOrCreate()
+
+// COMMAND ----------
+
+// DBTITLE 1,Define repository
+// Use the DeltaRepository to access data from Delta storage tables
+val repository = new DeltaRepository(rawDb, featureDb)
+
+// Use the SampleRepository to generate sample data
+// val repository = new SampleRepository(15)
```

Comment:
Should this still be in here? Or is it there so other people can try it out without needing a table set up?